Add on premise microk8s deployment guide (#5)
davidmirror-ops/flyte-the-hard-way: docs/on-premises/microk8s/02-install-configure-microk8s.md

# Install and Configure MikroK8s

> To deploy kubernetes on your ubuntu powered device install the microk8s snap.

Reviewer: ```suggestion
To deploy Kubernetes on your Ubuntu-powered device install the microk8s snap.
```

> This tutorial uses the 1.25 channel version
``` bash
sudo snap install microk8s --classic --channel=1.25
```

> To enable your current user seamless usage of microk8s command, run the following commands:

Reviewer: ```suggestion
To enable the `microk8s` command for your current user,  run the following commands:
```

``` bash
sudo usermod -a -G microk8s $USER
sudo chown -f -R $USER ~/.kube
newgrp microk8s
```
To validate the microk8s installation you can run the following command to display the current status of microk8s and enabled/disabled addons:
``` bash
microk8s status
```

> If this successfully displays the microk8s status, your are ready to operate via kubectl like shown in the following example:

Reviewer: ```suggestion
If it successfully displays the microk8s status, you are ready to operate via `kubectl` like shown in the following example:
```
